codesign/inputparse.go

```diff
@@ -203,14 +204,14 @@ func validateAndExpandProfilePaths(profilesList string) ([]string, error) {
 func listProfilesInDirectory(dir string) ([]string, error) {
 	exists, err := pathutil.IsDirExists(dir)
 	if err != nil {
-		return nil, fmt.Errorf("failed to check if provisioning profile path (%s) exists: %w", dir, err)
+		return nil, fmt.Errorf("failed to check if provisioning profile path exists (%s): %w", dir, err)
 	} else if !exists {
-		return nil, fmt.Errorf("please provide remote (https://) or local (file://) provisioning profile file paths with a scheme, or a local directory without a scheme: profile directory (%s) does not exist", dir)
+		return nil, fmt.Errorf("directory of provisioning profiles does not exist (%s)", dir)
```

Comment:
It would be useful to have a standardized way to add suggestions on how to solve issues to error strings.

Reply:
Fully agree and working on it, more improvements will be coming for the error messages.